geode-core/src/main/java/org/apache/geode/security/AuthInitialize.java

```diff
@@ -36,28 +33,31 @@
  */
 public interface AuthInitialize extends CacheCallback {
 
+  String SECURITY_USERNAME = "security-username";
+  String SECURITY_PASSWORD = "security-password";
+
   /**
    * Initialize the callback for a client/peer. This is invoked when a new connection from a
    * client/peer is created with the host.
    *
+   * Do not use these loggers, use log4j logger directly
+   *
```

Comment:
I find the wording in the javadoc confusing - we are saying use the init(systemLogger, securityLogger) method, but don't use the logger arguments in the AuthInitialize implementors.

I understand that we need to continue support any existing implementations, so maybe the  javadoc should indicate that new implementations should use log4j logger instead of the  logger parameters.